src/appleseedmaya/shadingnodemetadata.cpp

```diff
@@ -92,6 +95,13 @@ namespace
         defaultValue.push_back(v[1]);
         defaultValue.push_back(v[2]);
     }
+
+    void getFloat2Default(const asf::Dictionary& paramInfo, std::vector<double>& defaultValue)
```

Comment:
"get" methods usually return a result. Make "extract"?

Reply:
I followed @est77 suggestion here. Since it's not used anywhere else (for now at least), it's now inlined.